build.gradle

```diff
@@ -0,0 +1,179 @@
+buildscript {
+    ext {
+        corda_release_group = 'net.corda'
+        corda_release_version = '5.0-SNAPSHOT-WILLH'
+        corda_gradle_plugins_version = '4.0.42'
+        kotlin_version = '1.2.71'
+        junit_version = '4.12'
+        quasar_version = '0.7.10'
+        slf4j_version = '1.7.25'
+        log4j_version = '2.11.2'
+        corda_platform_version = '4'
+        tokens_release_group='com.r3.corda.lib.tokens'
```

Comment:
Is tokens used in this repo ? I'm not sure it should be a dependency of CIs but maybe it's used in some of the integration tests?

Reply:
Yeah it's used in the integration test that's all. We can remove the integration test if the dependency is more of an issue?

Reply:
Nah that's ok - was just wondering. We just need to remember to use `testCompile` in the workflows build gradle.